src/main/java/com/example/WorldOfAnimals/controllers/AnimalTypeController.java

```diff
@@ -0,0 +1,59 @@
+package com.example.WorldOfAnimals.controllers;
+
+import com.example.WorldOfAnimals.dto.AnimalTypeDTO;
+import com.example.WorldOfAnimals.exceptions.AnimalTypeNotFoundException;
+import com.example.WorldOfAnimals.services.AnimalTypeService;
+import lombok.AllArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/animalTypes")
+@AllArgsConstructor
+public class AnimalTypeController {
+
+    private final AnimalTypeService service;
+
+    @PostMapping
+    @ResponseStatus(HttpStatus.CREATED)
+    public void saveAnimalType(@RequestBody AnimalTypeDTO animalTypeDTO) {
```

Comment:
Need create new class AnimalTypeRequest with one field 'name', because in this RequestBody AnimalTypeDTO have 2 filed, but you can not use id when create new AnimalType

Reply:
done